x/reward/types/types.go

```diff
@@ -47,3 +47,16 @@ func (m Pool) Validate() error {
 
 	return nil
 }
+
+// ValidateBasic returns an error if the Refund is not valid; nil otherwise
+func (m Refund) ValidateBasic() error {
+	if err := sdk.VerifyAddressFormat(m.Payer); err != nil {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, sdkerrors.Wrap(err, "payer").Error())
+	}
+
+	if err := m.Fees.Validate(); err != nil {
```

Comment:
The `Validate` function prevents fees to be `0`. A validator could have included a refundable transaction with 0 fees in principle. That will raise an error in the refund ante handle, so it doesn't break anything later at least, and I'm ok with keeping `0` fee as disallowed. Some chains like Osmosis do use a min gas price of `0`.

Reply:
I'm taking this as an approval :)